CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 1.1.0 - 21 Apr 2020
+### Added
+* Custom file names for docker-compose files can be defined [#102](https://github.com/microsoft/vscode-docker/issues/102)
+* The experience for pushing Docker images has been revamped [#351](https://github.com/microsoft/vscode-docker/issues/351), [#1539](https://github.com/microsoft/vscode-docker/issues/1539), [#1595](https://github.com/microsoft/vscode-docker/issues/1595)
+* Extensibility model for registry providers has been improved [#147](https://github.com/microsoft/vscode-docker/issues/147)
+* Generic DockerV2 registries using OAuth can now be connected to in many cases [#869](https://github.com/microsoft/vscode-docker/issues/869)
+
```

Comment:
What about the new feature that we now react to Docker context changes? That includes context commands in the palette.

Reply:
Good idea, I'll add that.